libcloudforensics/providers/gcp/internal/monitoring.py

```diff
@@ -91,6 +91,90 @@ def ActiveServices(self, timeframe: int = 30) -> Dict[str, int]:
               ret[service] = int(val)
     return ret
 
+
+  def GetNetworkData(
+      self, instance_ids: Optional[List[str]] = None,
+      days: int = 2) -> List[Dict[str, Any]]:
```

Comment:
Is it possible that a user might want to specify a start and end date? Rather than just "the last N days."

Reply:
Its possible but will leave it to match the existing code patterns for now